oneplus/learn_views.py

```diff
@@ -236,7 +236,7 @@ def nextchallenge(request, state, user, participant):
             answerdate__gte=date.today()
         ).distinct('participant', 'question').count() + 1
 
-    points = " - %d point question" % _learnerstate.active_question.points
+    points = "%d points" % _learnerstate.active_question.points
```

Comment:
maybe account for 1 point? Django has 'pluralize' for templates . not sure if you can use that in this case.